src/main/java/me/coley/cafedude/io/AttributeWriter.java

```diff
@@ -252,11 +255,115 @@ public AttributeWriter(ClassFile clazz) {
 				case Constants.Attributes.SOURCE_ID:
 					break;
 				case Constants.Attributes.STACK_MAP_TABLE:
+					StackMapTableAttribute stackMapTable =
+						(StackMapTableAttribute) attribute;
+					writeStackMapTable(out, stackMapTable);
 					break;
 				default:
 					break;
 			}
 		}
 		return baos.toByteArray();
 	}
+	
+	private void writeVerificationType(
+		DataOutputStream out,
+		StackMapTableAttribute.TypeInfo type
+	) throws IOException {
+		if (type instanceof StackMapTableAttribute.TopVariableInfo) {
+			out.writeByte(0);
```

Comment:
May want to abstract this byte out into the info class. It'll get rid of a lot of this `else if` chaining